wagtail/tests/testapp/models.py

```diff
@@ -295,6 +297,18 @@ class EventPage(Page):
 ]
 
 
+class HeadCountRelatedModelUsingPK(models.Model):
+    """Related model that uses a custom primary key (pk) not id"""
+    custom_id = models.AutoField(primary_key=True)
```

Comment:
It would test more edge cases to use a non-integer primary key, here.

Reply:
`UUIDField(primary_key=True, default=uuid.uuid1)`